client/src/components/DownloadStarted.js

```diff
@@ -0,0 +1,144 @@
+import React, { Fragment } from 'react'
+import { Box, Grid, Heading, Text } from 'grommet'
+import { Button } from 'components/Button'
+import { Link } from 'components/Link'
+import { formatBytes } from 'helpers/formatBytes'
+import { config } from 'config'
+import { useResponsive } from 'hooks/useResponsive'
+import { downloadOptions } from 'config/downloadOptions'
+import { isProjectID } from 'helpers/isProjectID'
+import DownloadSVG from '../images/download-folder.svg'
+
+// View when the donwload should have been initiated
+export const DownloadStarted = ({
+  resource,
+  computedFile,
+  handleSelectFile
+}) => {
+  // open the file in a new tab
+  const { size: responsiveSize } = useResponsive()
+  const { size_in_bytes: size, download_url: href } = computedFile
+  const isProject = isProjectID(resource.scpca_id)
+  const startedText = isProject
+    ? 'Your download for the project should have started.'
+    : 'Your download for the sample should have started.'
+  const idText = `${isProject ? 'Project' : 'Sample'} ID: ${resource.scpca_id}`
+  const otherComputedFiles = resource.computed_files.filter(
+    (cf) => cf.id !== computedFile.id
+  )
+  const downloadOptionType = downloadOptions[computedFile.type]
+
+  const handleClick = (file) => {
```

Comment:
you can get rid of this for now and just call handleSelectFile directly
also, you could you add `() => {}` as the default for handleSelectFile in the props for the function